src/main/java/seedu/address/model/Model.java

```diff
@@ -50,7 +52,9 @@ public interface Model {
      */
     void setAddressBook(ReadOnlyAddressBook addressBook);
 
-    /** Returns the AddressBook */
+    /**
+     * Returns the AddressBook
```

Comment:
There seems to be a missing period `.` here.

Reply:
Ok. I made the changes.